lib/tests/streamlit/elements/video_test.py

```diff
@@ -19,17 +19,17 @@
 import numpy as np
 
 import streamlit as st
-from streamlit.runtime.media_file_manager import (
+from streamlit.runtime.media_file_storage import MediaFileStorageError
+from streamlit.runtime.memory_media_file_storage import (
     _calculate_file_id,
-    media_file_manager,
-    STATIC_MEDIA_ENDPOINT,
 )
+from streamlit.web.server.server import MEDIA_ENDPOINT
 from tests import testutil
 
 
 class VideoTest(testutil.DeltaGeneratorTestCase):
-    def test_st_video(self):
-        """Test st.video."""
```

Comment:
This was (another) monolithic "test all the things" test function that I split into smaller units.